python/lsst/ip/diffim/modelPsfMatch.py

```diff
@@ -49,6 +56,28 @@ class ModelPsfMatchConfig(pexConfig.Config):
         ),
         default="AL",
     )
+    doAutoPadPsf = pexConfig.Field(
+        dtype=bool,
+        doc=("Automatically pad science PSF to minimum dimensions appropriate for the kernel. "
+             "If the Science Psf is smaller than a the matching kernel + a minimum fraction. "
```

Comment:
remove either "a" or "the" and change + to "add" or "pad by" the sentence is a bit confusing on the first read. I read it as "If the Science Psf is smaller than (the matching kernel + a minimum fraction)"